linter.py

```diff
@@ -12,7 +14,14 @@
 class AutoLintOnTabSwitchListener(sublime_plugin.ViewEventListener):
     @classmethod
     def is_applicable(cls, settings):
-        return True
+        try:
+            user_settings = sublime.load_settings('SublimeLinter.sublime-settings')
+            auto_lint_setting = settings.get('SublimeLinter.linters.phpstan.auto_lint_on_tab_switch') or user_settings.get('linters')['phpstan']['auto_lint_on_tab_switch']
+            auto_lint = bool(auto_lint_setting)
+        except KeyError:
+            auto_lint = False
```

Comment:
I chose to use `False` as the default because that was the original behaviour for previous versions. It's quite a drastic change that not everyone might need enabled out of the box.